tests/test_json.nim

```diff
@@ -0,0 +1,22 @@
+import unittest
+import json
+
+import moustachu
+
+let objTemplate = "s={{string}},i={{int}},f={{float}}"
+
+suite "test json input":
+  test "from string template":
+    var j = newJObject()
+    j["string"] = newJString("hello")
+    j["int"] = newJInt(3)
+    j["float"] = newJFloat(3.14)
+    let expected_result = "s=hello,i=3,f=3.14"
+    check render(objTemplate, j) == expected_result
+  test "from file":
```

Comment:
minor: new line between the `test` blocks

Reply:
Will add